workspaces/simon-game/src/app/components/App.tsx

```diff
@@ -1,14 +1,21 @@
-import React from "react";
+import React, { useState } from "react";
 
 import { Box } from "./Box.tsx";
 import { playNote } from "../util/playNote.ts";
 import { config } from "../constants.ts";
 
 export function App() {
+  const [playerMoves, setPlayerMoves] = useState<number[]>([]);
   return (
     <div style={{ display: "flex", gap: 10 }}>
-      {config.boxes.map((box) => (
-        <Box color={box.color} onClick={() => playNote(box.frequency)} />
+      {config.boxes.map((box, index) => (
+        <Box
+          color={box.color}
+          onClick={() => {
+            playNote(box.frequency);
+            setPlayerMoves([...playerMoves, index]);
+          }}
+        />
```

Comment:
If you just want to show some data on the screen (great when a feature is in development). You can do render something like:

```tsx
<pre>{JSON.stringify(playerMoves, null, 2)}</pre>
```